game/ui/list.cc.orig

```diff
@@ -0,0 +1,249 @@
+#include "list.hh"
```

Comment:
There's still accidentally commited files left in.

Reply:
Removed